core/controllers/base.py

```diff
@@ -13,18 +13,20 @@
 # limitations under the License.
 
 """Base constants and handlers."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import division  # pylint: disable=import-only-modules
+from __future__ import print_function  # pylint: disable=import-only-modules
 
-import Cookie
 import base64
 import datetime
 import hmac
+import http.cookies
```

Comment:
Is this a module in python 2? I can't find a python2 documentation page that mentions it, and the codebase still needs to work in python 2...

If not, you may need to do something like what we do in core.platform.models (to choose the right import dynamically).

Reply:
No this is present in python 2 also. The test runs fine with http.cookies and also I tried importing it locally in Python 2 interpreter and it works